waypoint/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
```

Comment:
just call `poetry update` in this folder, and then `poetry lock` to make sure everything here is up to date